open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/kev/KevItem.java

```diff
@@ -27,6 +28,7 @@
 /**
  * Known Exploited Vulnerability.
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
```suggestion
```

Reply:
KEV does not disallow additional properties in schema so retained them: https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities_schema.json